README.md

```diff
@@ -323,6 +323,10 @@ as `onNext`, `onComplete`, or `cancel`.
 
 `take%%<n>%%<id>` : The driver should hang the test thread until it has received at least `n` items IN TOTAL, and then immediately cancel.
 
+`disconnect` : The driver should disconnect the client from the server.
+
+`resume` : The driver should resume the connection with the server.
```

Comment:
In most of the resumption related failure, Rsocket will generate the error, right? My server application will not be aware of those errors. So I might not able to express that in the DSL.
 
To handle such cases, I can add something like assertClientError, right? 

To express more fine grained control over resumption, the language need to be extended. In this commit, the target is to run a basic case of resumption